.gitignore

```diff
@@ -20,6 +20,8 @@ _build
 *.annot
 *.mj
 *.bak
+*.swp
+*.swo
```

Comment:
> since VIM is general enough, it is fine to ignore those temporary files

It is fine yeah, but if someone is using an IDE/editor and it lives file litter around (yes I use VIM too, and spacemacs) they should setup a user-profile gitignore to have those be ignored by default so that they do not need to submit PR's to every-single-project they work on.  ^.^;